pkg/dumper/dump_test.go

```diff
@@ -167,7 +167,28 @@ var _ = Describe("Dump", func() {
 		BeforeEach(func() {
 			logger = zap.NewNop()
 			upserter = &postgresfakes.FakeUpserter{}
-			r = strings.NewReader("[{},{},{}]")
+			r = strings.NewReader(`[
+				{
+					"DIRECTION": "N",
```

Comment:
So I was talking about this with Tay the other day and this as good a place as any to solicit thoughts: what do you think about codifying and building a common identifiers table(s) for all the stations, lines, IDs, etc...? We're in this weird spot where we're getting whipped around by MARTA's inconsistency with naming conventions (e.g., Dome/Omni, N vs. North, etc...)

I was talking it over informally the other day and was thinking it might useful for us to sit down and come up witha shared idea of how we want to refer to all the items in the system, and then categorize and trap all of MARTA's permutations and foibles in a common representation. This might manifest itself, practically, in a spreadsheet in the infohub project, and then each individual project implementing that alignment for their respective platform.

Also a non-zero chance I'm overthinking this, so I welcome all feedback/pushback on this idea :grimacing:

Reply:
I think that’s a great idea, and we could expose it as an API as well although I’m not sure if that’d be useful. A lot of that stuff is listed out in this PR in the file taxonomy.go! I’d be happy to migrate that into a google sheet or something similar, or to write a simple API that serves up that static data

Reply:
Yeah I think what you've got here is a great place to start - codifying based off that, and then getting that canonically into the infohub and maybe exposing a static API would be great! I'll go ahead and start building out the Ruby API client around that assumption and we can talk strategy for getting it all on the same page at the next meeting :sunglasses: